README.md

```diff
@@ -109,7 +106,7 @@ cm.onConnect(function() {
 
 The *onNextConnect* method queues a task (the callback) to run the next time the device connects. If the imp is already connected, the callback will be invoked immediatly.
 
-The callback method takes zero parameters.
+*The callback method takes zero parameters.*
 
 ```squirrel
 function poll() {
```

Comment:
probably also worth noting that in this example "sensor" is presumed to return a table, as the example will break if it doesn't.